### PR TITLE
Node v4/v6 removal and associated tasks

### PR DESCRIPTION
**Pull request by fgalan**
- [x] Remove Node v4 and v6 from Travis
- [x] Set Node version >=8 in packages.json 
- [x] Update logops to 2.0.0 **(at the end this has been deferred to issue https://github.com/telefonicaid/iotagent-manager/issues/160, check detail there)**
- [x] Update dependencies based in npm audit
- [x] Replace grunt by npm scripts

### CODE REVIEW

**Comment by fgalan** [comment]
After last fixes:

```
added 344 packages from 776 contributors and audited 956 packages in 17.028s
found 0 vulnerabilities
```

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,4 +1,16 @@
+- Set Nodejs 8.12.0 as minimum version in packages.json
 - Save cbHost in configuration
 - Upgrade Dockerfile to use NodeJS 8
 - Add: use PM2 in Dockerfile
-- Upgrade mongoose dependency to 4.13.14
+- Upgrade mongoose dependency from ~4.1.15 to 4.13.14
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE]: Note (check packages.json) that mongosee depenceny has not been updated in this PR. This line has been modified to include the "from" version to align with the style used by the next ones.

NTC (just informative)

**Comment by fgalan** [comment]
After last commit,  `npm install` shows:

```
added 334 packages from 771 contributors and audited 948 packages in 22.558s
found 0 vulnerabilities
```

File: lib/model/dbConn.js [hunk 2]
```diff
@@ -39,6 +39,8 @@ var mongoose = require('mongoose'),
         op: 'IoTAManager.DbConn'
     };
 
+mongoose.Promise = global.Promise;  // not including this causes DeprecationWarning
```
Comment (fgalan) [inline lib/model/dbConn.js]: This avoids warning:

```
	(node:53296) DeprecationWarning: Mongoose: mpromise (mongoose's default promise library) is deprecated, plug in your own promise library instead: http://mongoosejs.com/docs/promises.html
```

NTC (just informative)

**Comment by fgalan** [comment]
@cesarjorgemartinez given this PR changes the grunt intervace (i.e. "grunt test" is now "npm test") please provide LGTM to ensure PR is ready for merge.

**Comment by cesarjorgemartinez** [comment]
LGTM

**Pull request merged**